workbench-core/base/src/aws/helpers/dynamoDB/transactEdit.ts

```diff
@@ -57,6 +57,40 @@ class TransactEdit {
     return this;
   }
 
+  /**
+   * Add one or more delete request(s) to the command input. Use to delete one or more items from a DynamoDB Table.
+   *
+   * @param keys - list of object(s) of the primary key(s) of item(s) to delete
+   * @returns TransactEdit item with populated params
+   */
+  public addDeleteRequests(keys: { [key: string]: AttributeValue }[]): TransactEdit {
+    keys.forEach((key) => {
+      this.addDeleteRequest(key);
+    });
+    return this;
+  }
+
+  /**
+   * Add a single delete request to the command input. Use to delete a single item from a DynamoDB Table.
+   *
+   * @param key - object of the primary key of item to delete
+   * @returns TransactEdit item with populated params
+   */
+  public addDeleteRequest(key: { [key: string]: AttributeValue }): TransactEdit {
+    if (!this._params.TransactItems) {
+      throw new Error(
+        'TransactEdit<==need to initialize the TransactItems property before adding new request'
```

Comment:
Why do we have a `<==` in the error message?

Reply:
It's an arrow. The other error messages of this file also have them.